packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -832,8 +838,17 @@ export const prepareVaultManagerKit = (
             /** @type {Array<[Vault, { collateralAmount: Amount<'nat'>, debtAmount:  Amount<'nat'>}]>} */
             for (const [vault, balance] of bestToWorst) {
               const { collateralAmount: vCollat, debtAmount } = balance;
+              const vaultPenalty = ceilMultiplyBy(debtAmount, penaltyRate);
```

Comment:
This value is used only to calculate `collatPostDebt`. Consider factoring out a helper function to to the math. It looks like the change above could uses the same.